tests/Rector/Class_/RenameClassRector/RenameClassRectorTest.php

```diff
@@ -44,6 +57,12 @@ protected function getRectorsWithConfiguration(): array
                 Manual_Twig_Filter::class => TwigFilter::class,
                 'Twig_AbstractManualExtension' => AbstractManualExtension::class,
                 'Twig_Extension_Sandbox' => 'Twig\Extension\SandboxExtension',
+                'MyNamespace\\MyClass' => 'MyNewNamespace\\MyNewClass',
```

Comment:
```suggestion
                'MyNamespace\MyClass' => 'MyNewNamespace\MyNewClass',
```

Reply:
\ is enough, same for bellow

There should coding standard for this if not regular expression :)